Komodo/Assets/Packages/KomodoCore/Runtime/Scripts/RuntimeSession/Managers/UIManager.cs

```diff
@@ -589,26 +589,48 @@ public bool IsReady ()
         public void EnableHightCalibrationButtons() 
         {
             //KomodoMenu -> Panels -> SettingsMenu
-            GameObject SettingsMenu = menu.transform.Find("Panels").transform.Find("SettingsMenu").gameObject;
+            GameObject settingsMenu = menu.transform.Find("Panels").transform.Find("SettingsMenu").gameObject;
 
             //KomodoMenu -> Panels -> SettingsMenu -> HeightCalibration
-            GameObject heightCalibration = SettingsMenu.transform.Find("HeightCalibration").gameObject;
+            GameObject heightCalibration = settingsMenu.transform.Find("HeightCalibration").gameObject;
 
             //KomodoMenu -> Panels -> SettingsMenu -> NotCalibrating -> CalibrateHeightButton
-            GameObject calibrationButtons = SettingsMenu.transform.Find("NotCalibrating").transform.Find("CalibrateHeightButton").gameObject;
+            GameObject calibrationButtons = settingsMenu.transform.Find("NotCalibrating").transform.Find("CalibrateHeightButton").gameObject;
 
             //KomodoMenu -> Panels -> SettingsMenu -> NotCalibrating -> ManuallyAdjustHeight
-            GameObject manuallyAdjustHeight = SettingsMenu.transform.Find("NotCalibrating").transform.Find("ManuallyAdjustHeight").gameObject;
+            GameObject manuallyAdjustHeight = settingsMenu.transform.Find("NotCalibrating").transform.Find("ManuallyAdjustHeight").gameObject;
 
             //createMenu.gameObject.SetActive(false);
             heightCalibration.gameObject.SetActive(true);
             calibrationButtons.gameObject.SetActive(true);
             manuallyAdjustHeight.gameObject.SetActive(true);
         }
 
-        public void EnableCreateMenu() {
+        public void EnableCreateMenu()
+        {
             GameObject createMenu = menu.transform.Find("Tabs").transform.Find("Create").gameObject;
             createMenu.gameObject.SetActive(true);
         }
+
+        public void DisableInstructorMenuButton()
+        {
+
+            GameObject settingsMenu = menu.transform.Find("Panels").transform.Find("SettingsMenu").gameObject;
```

Comment:
(Future) Try to locate component to find an object, rather than use the name of the object